releases/release-7.6.0.md

```diff
@@ -0,0 +1,288 @@
+
+    For more information, see [documentation](/sql-statements/sql-statement-create-index.md#multi-valued-indexes).
+
+### Reliability
+
+* 跨数据库绑定执行计划 [#48875](https://github.com/pingcap/tidb/issues/48875) @[qw4990](https://github.com/qw4990) **tw@Oreoxmt** <!--1613-->
+
+    我们看到越来越多的用户运行 TiDB 支撑其 SaaS 平台。SaaS 平台的一种普遍的建模方式，是把平台上每个租户的数据存入不同的“数据库”，而业务逻辑完全相同。这样我们能看到上百个数据库拥有相同的表和索引定义，运行类似的 SQL 语句。在这种场景下，当我们要对一条 SQL 语句的执行计划进行绑定(SQL Binding)，这条绑定通常对运行在其他数据库的 SQL 也有帮助。
+
+    针对这种应用场景，TiDB 引入了 "通用绑定" (Universal SQL Binding) 进行跨数据库绑定。 用一个执行计划绑定匹配模式相同的 SQL，即使他们运行在不同数据库上。比如，当我们创建了下面绑定，无论 `t1` 和 `t2` 位于哪个数据库，TiDB 都会尝试用此绑定的规则来生成执行计划，而不需要为每个数据库上的 SQL 单独创建。
+
+    ```sql
+    CREATE GLOBAL UNIVERSAL BINDING for
+        SELECT * FROM t1, t2 WHERE t1.id = t2.id
+    USING
+        SELECT /*+ merge_join(t1, t2) */ * FROM t1, t2 WHERE t1.id = t2.id;
+    ```
+
+    "通用绑定" 还能够有效缓解 Saas 场景中一个常见问题。SaaS 客户的数据及负载可能因为客户业务本身的需要而剧烈变化，通常 SaaS 服务商无法预测。在这个场景下，由于统计信息无法及时更新，数据量由小及大的快速变化经常会引发 SQL 性能问题。为解决这个问题，SaaS 服务商可以用 "通用绑定" 固定大数据量客户已经验证过的执行计划，所有客户的执行计划都会被固定。 针对 SaaS 平台，这一功能的引入为客户带来了显著的便利和体验提升。
+
+    由于 "通用绑定" 会引入一个很小的系统开销（小于1%）， 默认将其关闭。 有需要的用户通过设置变量 [`tidb_opt_enable_universal_binding`]() 为 `YES` 开启。
+
+    更多信息，请参考[用户文档](/sql-plan-management.md)。
+
+### Availability
+
+* Support the proxy component TiProxy (Experimental) [#413](https://github.com/pingcap/tiproxy/issues/413) @[djshow832](https://github.com/djshow832) @[xhebox](https://github.com/xhebox) **tw@ran-huang** <!--1596-->
+
+    TiProxy is the official proxy component of TiDB, located between the client and TiDB server. It provides load balancing and connection persistence functions for TiDB, making the workload of the TiDB cluster more balanced and not affecting user access to the database during maintenance operations.
+
+    - During maintenance operations such as rolling restarts, rolling upgrades, and scaling-in in a TiDB cluster, changes occur in the TiDB servers which result in interruptions in connections between clients and the TiDB servers. By using TiProxy, connections can be smoothly migrated to other TiDB servers during these maintenance operations so that clients are not affected.
+     - Client connections to a TiDB server cannot be dynamically migrated to other TiDB servers. When the workload of multiple TiDB servers is unbalanced, it might result in a situation where the overall cluster resources are sufficient, but certain TiDB servers experience resource exhaustion leading to a significant increase in latency. To address this issue, TiProxy provides connection dynamic migration, which allows connections to be migrated from one TiDB server to another without any impact on the clients, thereby achieving load balancing for the TiDB cluster.
```

Comment:
```suggestion
    - During maintenance operations such as rolling restarts, rolling upgrades, and scaling-in in a TiDB cluster, changes occur in the TiDB servers which result in interruptions in connections between clients and the TiDB servers. By using TiProxy, connections can be smoothly migrated to other TiDB servers during these maintenance operations so that clients are not affected.
    - Client connections to a TiDB server cannot be dynamically migrated to other TiDB servers. When the workload of multiple TiDB servers is unbalanced, it might result in a situation where the overall cluster resources are sufficient, but certain TiDB servers experience resource exhaustion leading to a significant increase in latency. To address this issue, TiProxy provides connection dynamic migration, which allows connections to be migrated from one TiDB server to another without any impact on the clients, thereby achieving load balancing for the TiDB cluster.
```